app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>RailsRecipeApp</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_importmap_tags %>
+  </head>
+
+  <body>
+  <% if current_user %>
+  <nav class="navbar navbar-expand-lg navbar-dark bg-primary">
+    <div class="container">
+      <a class="navbar-brand" href="#">Recipes</a>
+      <a class="navbar-brand"><%= current_user.name %></a>
+        <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+          <div class="navbar-nav">
+            <%= link_to 'Foods' , foods_url, class: "nav-item nav-link active" %>
+            <%= link_to 'Recipes' , recipes_url, class: "nav-item nav-link active" %>
+            <%= link_to 'Inventories' , inventories_url, class: "nav-item nav-link active" %>
+            <%= link_to "Logout", destroy_user_session_path, class: 'nav-item nav-link justify-content-end', data: { turbo_method: :delete } %>
+          </div>
```

Comment:
- [ ] Please add a Public recipe list that matches [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/recipe-app/images/public_recipe_index.png?token=GHSAT0AAAAAABROFRKQNYWXEBQHL2IJOEJ4YYSI5TA) 
 Users should see only their own recipes in one page, and there should be another public recipes page where all public recipes are rendered. 

- [ ] Since you are a team of 3 members, you should have a **Inventory shopping list:**

* The generated shopping list page is matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/inventory_shopping_list.png?raw=true)
* Shows the list of food that is missing by comparing the food in the recipe with the food in the inventory.
* Counts the total food items and total price of the missing food.Inventory shopping list:



- [ ] In order to follow a DRY rule, please move the navbar to a partial and render it in `application.html `, this way you can have more manageable code and follow best practices.

Reply:
@ShoiraTa I have added the top part to include public_recipe but the other requirements are not part of our scope of work. I would need more time to implement that